package-lock.json

```diff
@@ -9,437 +9,9 @@
         "starknetkit": "^2.3.0"
       },
       "devDependencies": {
-        "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
         "eslint-plugin-import": "^2.29.1"
       }
     },
-    "node_modules/@ampproject/remapping": {
```

Comment:
why did you change this file, in your task you don't need to install any new dependency